lib/consolidate.js

```diff
@@ -1034,6 +1034,9 @@ exports.nunjucks.render = function(str, options, fn) {
   return promisify(fn, function (fn) {
     try {
       var engine = requires.nunjucks || (requires.nunjucks = require('nunjucks'));
+      if (options.views) engine.configure(options.views);
+      if (options.settings && options.settings.views) engine.configure(options.settings.views);
```

Comment:
`options.settings` is what we get from express during render call, so I think this is the option to keep. I've handled `options.view` just to pass include test. I will remove `options.view` and adapt test case.